Source/FullDateCollectionViewCell.swift

```diff
@@ -14,8 +14,12 @@ class FullDateCollectionViewCell: UICollectionViewCell {
     var numberLabel: UILabel!
     var darkColor = UIColor(red: 0, green: 22.0/255.0, blue: 39.0/255.0, alpha: 1)
     var highlightColor = UIColor(red: 0/255.0, green: 199.0/255.0, blue: 194.0/255.0, alpha: 1)
+    var normalColor = UIColor.white
 
     override init(frame: CGRect) {
+        if #available(iOS 13.0, *) {
+            normalColor = UIColor.secondarySystemGroupedBackground
+        }
```

Comment:
Please make the colors of this cell customizable too instead of harcoding their values

Reply:
I agree with you.
I was a bit confused, had the "populateItem" function (line 57) passed "normalColor" color.